src/sentry/conf/server.py

```diff
@@ -653,6 +654,7 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     ),
     Queue("unmerge", routing_key="unmerge"),
     Queue("update", routing_key="update"),
+    Queue("profiles.process", routing_key="profiles.process"),
```

Comment:
I think you will need to let ops know about this queue for it to get processed

Reply:
Yes, it's in the works already https://getsentry.atlassian.net/browse/OPS-1475 .